src/renderer/components/+helm-releases/release-menu.tsx

```diff
@@ -78,7 +78,7 @@ class NonInjectedHelmReleaseMenu extends React.Component<HelmReleaseMenuProps &
 
     return (
       <MenuActions
-        id="menu-actions-for-release-menu"
+        id={`menu-actions-for-release-menu-for-${release.name}`}
```

Comment:
Why not `release.getId()` here?

Reply:
That's even better. Missed that it existed. Changed.